dir.props

```diff
@@ -53,10 +53,10 @@
 
     <!-- Packaging configuration -->
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)pkg/Microsoft.NETCore.Platforms/runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(PackagesDir)Microsoft.NETCore.Platforms/1.0.1-rc2-23712/runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
It's hard coded a couple lines below this at line 70.  We could simply share that string.

I expect it to also be included in the buildtools package, its not right now (I missed including in the buildtools nuspec) so rather than wait for that fix we hardcoded it for now.